components/constellation/constellation.rs

```diff
@@ -914,6 +931,15 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
         }
     }
 
+    fn handle_request_from_filemanager(&mut self, message: FileManagerMsg) {
+        match message {
+            FileManagerMsg::OpenFileSelectDialog(patterns, multiple_files, sender) => {
+                let msg = EmbedderMsg::GetSelectedFiles(patterns, multiple_files, sender);
+                self.embedder_proxy.send(msg);
+            }
+        }
+    }
```

Comment:
If we are forwarding a message, maybe we want to keep the same name.

Also, I talked to @cbrewster and we agree that maybe it's better to give access to the EmbedderProxy to each script thread.